### PR TITLE
on-prem-installer-2.6-rel

### PR DESCRIPTION
**Pull request by resmaa**
**|Palette Release|On-Premises Installer Version|**
-|2.6|2.0.2| - Removed
+|2.6|2.1.0| - Added

**|Version|URL|Info|**
-|2.0.2|hubble-installer-202-10152021.ova|Oct 15 2021| -Removed 
+|2.1.0|hubble-installer-210.ova|Jun 1 2022 (Cert Manager change to 1.8)| - Added

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/751#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Siboney1 — APPROVED** [review]

**Pull request merged**
